geps/gep-2257/index.md

```diff
@@ -119,4 +195,7 @@ criteria:
 
 - Have a custom CEL validator for GEP-2257 Duration fields.
 
-- Have support for GEP-2257 Durations in standard Kubernetes libraries.
+The previous graduation criterion of "Have support for GEP-2257 Durations in
+standard Kubernetes libraries" has been removed. Work is in progress to add
+such parsers to `kube-rs`, `client-go`, and `client-python`, but it is not
+necessary to gate the graduation of GEP-2257 on this work.
```

Comment:
IMO, it's not necessary to complete these parsers before the GEP freeze in 1 week, but it is necessary to complete these parsers before the end of the API refinement phase (likely +4 weeks per https://gateway-api.sigs.k8s.io/contributing/release-cycle/#3-api-refinement-and-documentation). We should definitely clean up this guidance before our next release.

Interested in what others think here though:
/cc @youngnick @mlavacca

Reply:
Agreed.

Reply:
Sounds good from here, too.

Reply:
I agree as well